src/app/fyle/personal-cards/personal-cards.page.ts

```
@@ -75,15 +76,15 @@ export class PersonalCardsPage implements OnInit, AfterViewInit {
 
   linkedAccountsCount$: Observable<number>;
 
-  linkedAccounts$: Observable<PersonalCard[]>;
+  linkedAccounts$: Observable<PlatformPersonalCard[]>;
 
   loadCardData$: BehaviorSubject<{}> = new BehaviorSubject({});
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)